linux_server/authorized_identities.go

```diff
@@ -53,7 +54,7 @@ func (i *PubKeyIdentity) Verify(genericCandidate interface{}, base64Conversation
 			return nil, fmt.Errorf("unsupported signature algorithm '%s' for %T", unvalidatedToken.Method.Alg(), i)
 		},
 			jwt.WithIssuer(i.username),
-			jwt.WithSubject("ssh3"),
+			jwt.WithSubject("github.com/francoismichel/ssh3"),
```

Comment:
same as my above comment, if another implem appears, we would not we interoperable, so I would keep `ssh3` for now